shard.yml

```diff
@@ -32,8 +32,9 @@ dependencies:
     github: mperham/sidekiq.cr
     branch: master
 
-  twitter:
-    github: veelenga/twitter.cr
+  twitter-crystal:
+    github: sferik/twitter-crystal
+    branch: master
```

Comment:
I think we should use a fixed version/commit to avoid breaking builds. We can ask them to release a version with the current master branch.

Reply:
But [is locked](https://github.com/crystal-community/crystal-ann/blob/b664f24a138f0391794c70f6fa75cb3ba800c002/shard.lock#L125) in `shard.lock`...

Reply:
I'm not sure but as I see a ```shards update``` to update another shards could fetch master's branch of this shard too.